css/styles.css

```
@@ -0,0 +1,1040 @@
+    filter: hue-rotate(10deg) saturate(380%) brightness(130%);
+    /* animation: bounce 2s infinite; */
+    /* filter: hue-rotate(6deg) saturate(50%) brightness(200%) contrast(30%) sepia(30%); */
+}
+
+.trebek::before {
+    content: '😎';
+    position: absolute;
+    left: 50%;
+    transform: translateX(-50%);
+    z-index: 10;
+    transform: scale(1.2);
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

transform is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)